README.md

```diff
@@ -1,16 +1,16 @@
 # RobotCodeOffSeason2024
 
-TRIGON 5990's robot code for the 2024 off-season robot, "PLACEHOLDER".
+TRIGON 5990's robot code for the 2024 off-season robot, "Killshon".
 
 # External Programs
 
 All the "Advantage Scope" related files (CAD glTF files, layouts, etc.) can be
-found [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)   
+found [here](https://drive.google.com/drive/folders/1mMYOdDzlApMwTz2Wo5Z5GBv4ffCnPAjh?usp=drive_link)   
```

Comment:
Layout name is still tragedy.
And I also just noticed there's no comma in the end of the sentence here.